requirements.txt

```diff
@@ -1 +1,3 @@
 python-dateutil
+ovos-utils~=0.0, >=0.0.38
+ovos_workshop~=0.0, >=0.0.15
```

Comment:
I'd like to hold off on merging this until we've had a chance to bake in these updates for a few days.

Reply:
these are the latest stable versions, and the minimum versions for the refactored imports

Reply:
![image](https://github.com/OpenVoiceOS/ovos-skill-easter-eggs/assets/30268971/96e3a997-5d8a-485a-b75a-87391b45aa3a)

It's a holiday weekend for a lot of people and I don't think they're eager to have to fix their assistant because a skill dragged a new dependency

Reply:
also note without this the skill wont work with ovos-utils 0.1.0   (core is pinning 0.0.38)

but utils will just log deprecation warnings until then, skill keeps working, this is not critical